Language/Classes/Instance_Methods/Method_noSuchMethod/invocation_A01_t01.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A noSuchMethod forwarder is a concrete member of C with the
+/// signature taken from the interface of C, and with the same default value for
+/// each optional parameter. It can be invoked in an ordinary invocation and in
+/// a superinvocation, and when m is a method it can be closurized using a
+/// property extraction.
+///
+/// @description Checks that `noSuchMethod` forwarder can be invoked in an
+/// ordinary invocation and in a superinvocation
+/// @author sgrekhov22@gmail.com
+
+import "../../../../Utils/expect.dart";
+
+mixin class A {
+  int m1(int v, [covariant String s = "s1"]);
+
+  int m2(int v, {covariant String s = "s2"});
+
+  dynamic noSuchMethod(Invocation inv) {
+    return 42;
+  }
+}
+
+class C1 extends A {
+  test() {
+    Expect.equals(42, super.m1(1, "1"));
+    Expect.equals(42, super.m2(2, s: "2"));
+    Expect.equals(42, this.m1(1, "1"));
+    Expect.equals(42, this.m2(2, s: "2"));
+  }
+}
+
+class C2 with A {
+  test() {
+    Expect.equals(42, this.m1(1, "1"));
+    Expect.equals(42, this.m2(2, s: "2"));
```

Comment:
The reason why we can't call `super.` in this case would be that the superclass `Object with A` is abstract (a mixin application used as a superclass in a class declaration is always abstract), so there is no noSuchMethod forwarder for `m1` or `m2`.

It would probably be helpful for future readers of this code if there's a comment to that effect.

Also: Did some other test cover the case where `super.m1(...)` should be an error because there is no noSuchMethod forwarder because it's being looked up in a mixin application?